### PR TITLE
[SD-3021],[SD-3743],[SD-3776] An empty text editor box returns empty markup causing grief for validation

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: server/apps/prepopulate/app_prepopulate_data.json [hunk 1]
```diff
@@ -244,7 +244,13 @@
...
 		            "allowed": [1, 2, 3, 4, 5]
-		        }
+		        },
+                "body_html": {
+                    "type": "string",
+                    "required": true,
+                    "nullable": false,
+                    "empty": false
+                }
 		    }
```
Comment (sivakuna-aap) [inline server/apps/prepopulate/app_prepopulate_data.json]: I think this validation is required for correct also.

Comment (marwoodandrew) [inline server/apps/prepopulate/app_prepopulate_data.json]: Yes, but this update is just so the test triggers a validation error

Comment (sivakuna-aap) [inline server/apps/prepopulate/app_prepopulate_data.json]: Aha!!! :+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**